interface/modules/zend_modules/config/autoload/global.php

```diff
@@ -89,14 +89,17 @@
 
     $dbh = null; // Close pdo connection
 }
+// sites/<site_id>/sqlconf.php stores the database connection settings into a global sqlconf variable
+// we will use that instead of the individual globals set previously.
+$sqlConf = $GLOBALS['sqlconf'] ?? ['dbase' => '', 'host' => '', 'login' => '', 'pass' => '', 'port' => ''];
```

Comment:
@sjpadgett or @bradymiller do you see anything wrong with this approach?  We were losing database access when zend modules were called within an API request.